### PR TITLE
dotnet folder structure

### PR DESCRIPTION
**Pull request by DilmurodMak**
dotnet folder structure created #5  [Create dotnet strucure](https://github.com/benchpress-team/benchpress/issues/5)

### CODE REVIEW

File: framework/dotnet/BenchPress.Module/BenchPress.Module.csproj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (omusavi) [inline framework/dotnet/BenchPress.Module/BenchPress.Module.csproj:1]: This folder will be for Powershell so we dont need the csproj or program.cs file

**Review by omusavi — APPROVED** [review]
Let's keep the generator code in the repo for now, just ignore it from our current structure

**Comment by omusavi** [comment]
I went ahead and added this manually, btu in the future if you put "Closes #XXX" it will automatically link the issue to the PR and auto-close. @ayomustapha lets make sure that text is in the PR template

**Comment by DilmurodMak** [comment]
> 

Will do.

**Review by polatengin — APPROVED** [review]
it looks good to me, thanks 👍

**Pull request merged**